matplotcheck/raster.py

```diff
@@ -57,14 +57,15 @@ def assert_colorbar_range(self, crange):
             ), "Colorbar maximum is not expected value:{0}".format(crange[1])
 
     def _which_label(self, label, all_label_options):
```

Comment:
let's rename `all_label_options` to `expected_labels`. And let's make that a cnvention in MPC where - whenever we are using parameters that ask for expected values we name them `expected_name_of_thing_here`. this will make the code generally a lot easier to follow!!

Reply:
let's also rename this helper -- > `_check_label_matches_expected`
and have this return a boolean of True if it matches and false if it doesnt